Cargo.toml

```diff
@@ -8,7 +8,7 @@ description = "A Rust implementation of Olm and Megolm"
 repository = "https://github.com/matrix-org/vodozemac"
 version = "0.6.0"
 edition = "2021"
-keywords = ["matrix", "chat", "messaging", "olm", "end-to-end-encryption"]
+keywords = ["matrix", "chat", "messaging", "olm"]
```

Comment:
```suggestion
keywords = ["matrix", "chat", "messaging", "olm", "megolm", "double ratchet", "encryption", "E2EE", "cryptography"]
```

Reply:
Should we add a bunch more relevant ones that are < 20 chars while we're at it?

Reply:
You can have at most 5 keywords per crate. :face_exhaling: